File: det.py
import numpy as np
n_array=np.array([[55,35,25];
		   [44,32,3];
		   [10,40,70]];
print("Numpy Matrix is")
print(n_array)
det=np.linalg.det(n_array)
print("det of 3*3 matrix")
print(int det)

File: det.m
% Creating a 3x3 matrix
n_array = [55, 25, 15;
           30, 44, 2;
           11, 45, 77];

% Displaying the matrix
disp("Numpy Matrix is:");
disp(n_array);

% Calculating the determinant of the matrix
det = det(n_array);

disp("\nDeterminant of given 3x3 square matrix:");
disp(det);